### PR TITLE
Put student pages under /students

### PR DESCRIPTION
**Pull request by differentmatt**

### CODE REVIEW

File: app/core/Router.coffee [hunk 1]
```diff
@@ -72,15 +72,15 @@ module.exports = class CocoRouter extends Backbone.Router
     'contribute/diplomat': go('contribute/DiplomatView')
     'contribute/scribe': go('contribute/ScribeView')
 
-    'courses': go('courses/CoursesView')
-    'Courses': go('courses/CoursesView')
-    'courses/students': redirect('/courses')
+    'courses': redirect('/students') # Redirected 9/3/16
+    'Courses': redirect('/students') # Redirected 9/3/16
```
Comment (sderickson) [inline app/core/Router.coffee]: Testing on localhost, if I go to `/courses?_cc=test`, the query parameter is lost in the redirect. Looks like `redirect` needs to be set up to keep the query string.

Comment (sderickson) [inline app/core/Router.coffee]: Also it's also weird that if you don't have the query param included, CoursesView (`/students`) shows the error screen. Shouldn't `/students` look the same if you have no role set, regardless of GET query parameters?

https://github.com/codecombat/codecombat/blob/master/app/views/courses/CoursesView.coffee#L70-L71

**Comment by sderickson** [comment]
Looks good other than lost classroom codes on redirects

File: app/core/Router.coffee (hunk 1 above)
Comment (differentmatt) [inline app/core/Router.coffee]: Yikes, good catch on the redirects.
You can join a class and become a student with a query param, so I think that's why we delay showing an error screen there.
https://github.com/codecombat/codecombat/blob/students-routing/app/views/courses/CoursesView.coffee#L170

**Pull request closed without merging**